network-api/networkapi/wagtailpages/views.py

```diff
@@ -22,3 +23,15 @@ def custom404_view(request, exception):
     else:
         html = render(request, '404.html')
         return HttpResponseNotFound(html.content)
+
+
+def redirect_to_initiatives(request, subpath):
+    lang = request.LANGUAGE_CODE
+    translation.activate(lang)
+    request.session[translation.LANGUAGE_SESSION_KEY] = lang
+    query_string = ""
+
+    if request.META['QUERY_STRING']:
+        query_string = "?%s" % request.META['QUERY_STRING']
```

Comment:
This is a legacy string templating solution from Python 2.7 and is highly discouraged for 3.x

Instead, f-strings are the recommended way to composite strings now:

```suggestion
        query_string = f'?{request.META['QUERY_STRING']}'
```